### PR TITLE
[Feature #4032] Add Redis Connector modules 

### LINKED ISSUE #4032: [Feature] Add Redis Connector modules
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/eventmesh/issues?q=is%3Aissue) and found no similar issues.

### Feature Request

Add redis source connector module and sink connector module.

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by fabian4**
Fixes #4032.

### Motivation

*Explain the content here.*
*Explain why you want to make the changes and what problem you're trying to solve.*

### Modifications

*Describe the modifications you've done.*

### Documentation

- Does this pull request introduce a new feature? (yes / no)
- If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
- If a feature is not applicable for documentation, explain why?
- If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

**Review by mxsm — COMMENTED** [review]
@fabian4 Please check code style and CI not pass

File: eventmesh-connectors/eventmesh-connector-redis/src/main/java/org/apache/eventmesh/connector/redis/server/RedisConnectServer.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        RedisServerConfig serverConfig = ConfigUtil.parse(RedisServerConfig.class, "server-config.yml");
+
+        if (serverConfig.isSourceEnable()) {
+            Application kafkaSourceApp = new Application();
+            kafkaSourceApp.run(RedisSourceConnector.class);
+        }
+
+        if (serverConfig.isSinkEnable()) {
+            Application kafkaSinkApp = new Application();
+            kafkaSinkApp.run(RedisSinkConnector.class);
```
Comment (Pil0tXia) [inline eventmesh-connectors/eventmesh-connector-redis/src/main/java/org/apache/eventmesh/connector/redis/server/RedisConnectServer.java:42]: I suggest a naming change for `kafkaSourceApp` and `kafkaSinkApp`.

**Review by Pil0tXia — CHANGES_REQUESTED** [review]
Thank you for your contribution.

File: eventmesh-connectors/eventmesh-connector-redis/src/main/resources/sink-config.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+pubSubConfig:
+    meshAddress: 127.0.0.1:10000
+    subject: TopicTest
+    idc: FT
+    env: PRD
+    group: kafkaSink
```
Comment (Pil0tXia) [inline eventmesh-connectors/eventmesh-connector-redis/src/main/resources/sink-config.yml:23]: and here.

**Comment by fabian4** [comment]
The data encoding and decoding still in progress.

**Comment by Alonexc** [comment]
image
please check checkstyle.

**Comment by fabian4** [comment]
> image please check checkstyle.

Just fixed it.

BTW is there any way to check it automatically in my local IDEA. How to make it import the dependancy with the order we need as the rules said.

**Comment by Pil0tXia** [comment]
@fabian4 Please see https://eventmesh.apache.org/zh/community/contribute/contribute

**Review by mxsm — APPROVED** [review]
LGTM

**Review by xwm1992 — APPROVED** [review]

**Pull request merged**